src/components/TooltipContainer/TooltipContainer.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { TooltipContainer } from './TooltipContainer';
+
+describe('TooltipContainer component', () => {
+  it('renders visible tooltip when property visible is true', async () => {
+    //Arrange
+    const visible = true;
+    // Act
+    const { container } = render(<TooltipContainer visible={visible}></TooltipContainer>);
+
+    // Assert
+    await waitFor(() => {
+      expect(container.querySelector('.dp-tooltip-container .hidden')).not.toBeInTheDocument();
```

Comment:
Despite tooltip won't have the hidden class (what is true), maybe it could add another expect to validate that tooltip is showed:
`expect(container.querySelector('.dp-tooltip-container')).toBeInTheDocument();`